pkg/apputils/utils.go

```diff
@@ -173,8 +177,8 @@ func getGitClients(ctx context.Context, url, configUrl, namespace string, isHelm
 	return appClient, configClient, authsvc.GetGitProvider(), nil
 }
 
-func getAuthService(ctx context.Context, providerUrl string) (auth.AuthService, error) {
-	gitProvider, providerErr := auth.GetGitProvider(ctx, providerUrl)
+func getAuthService(ctx context.Context, normalizedUrl gitproviders.NormalizedRepoURL) (auth.AuthService, error) {
```

Comment:
:100: We need to start passing the `NormalizedRepoURL` instead of the original `string`.